todo_app/templates/index.html

```diff
@@ -11,8 +11,15 @@ <h1 class="display-4">To-Do App</h1>
     <div class="col-md-12">
       <h2>Items</h2>
       <ul class="list-group mb-4">
-        <!-- List the todo items here -->
+        {% for item in items %}
+          <li>{{ item.title }}</li>
```

Comment:
You might want to consider adding the status (especially as we'll be using it in the next exercise)